bin.src/oods.py

```diff
@@ -84,4 +83,10 @@ async def gather_tasks(config):
 
     LOGGER.info("***** OODS starting...")
 
+    if "archiver" in oods_config:
+        name = oods_config["archiver"]["name"]
+    else:
+        name = "unknown"
```

Comment:
Since the configuration validation is optional and disabled by default, you may consider using
```
try:
    name = oods_config["archiver"]["name"]
except KeyError:
    name = 'unknown'
```
instead. With the current approach, you'll end up with unhandled `KeyError` exception if `name` is not defined in `oods_config["archiver"]`.

Reply:
Fixed